sender.go

```diff
@@ -110,6 +110,15 @@ var ErrResponseUnexpected error = &SendError{
 	clogMsg:   "Response doesn't match expected values.",
 }
 
+// ErrResponseUnexpectedUnlogged should be returned when channel response doesn't match what we expect and unlogged
```

Comment:
// ErrResponseUnexpectedUnlogged is same as ErrResponseUnexpected without logging so better for channels where response check is user configured.